sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.
+
+For example, this includes:
+
+* Discussions on all levels from application development, application testing environments, and applications running in production
+* Discussion of how to define and run apps in Kubernetes
```

Comment:
How about:

* Discussion of how to define, develop, and run applications in Kubernetes

(Or maybe breaking this out into a separate line. As I read this charter, it seems that discussing the actual development workflow(s) for building apps isn't explicit. To me, the actual development of applications (not just the definition and how to run them) is what spawned some of the current charter conversation.)

Reply:
+1